lib/modules/datasource/deb/readme.md

```diff
@@ -1,7 +1,13 @@
-The Debian datasource enables Renovate to update packages from Debian repositories. It is ideal for projects that depend on Debian-based systems or distributions. You will need to combine Debian datasource with [regex managers](../../manager/regex/index.md) to update dependencies.
+Renovate uses the Debian datasource to update packages from Debian repositories.
+The `debian` datasource is meant for projects that:
+
+- depend on Debian-based systems, or
+- depend on Debian-based distributions, like Ubuntu
+  By default, Renovate does not detect Debian dependencies.
+  For Renovate to update dependencies, you must combine the Debian datasource with [regex managers](../../manager/regex/index.md).
 
 **Registry URL**
```

Comment:
```suggestion
## Set URL when using a Debian repository
```

This section needs a heading.

The `**Example**` bold text in the same section should be changed to a sub-heading: `### Example`.